pycaret/internal/display/__init__.py

```diff
@@ -1,5 +1,4 @@
-from pycaret.utils import enable_colab  # Imported in other modules - do not remove
-
+from ...utils.utils import enable_colab
 from .display import CommonDisplay, DummyDisplay
 
-__all__ = ["CommonDisplay", "DummyDisplay", "enable_colab"]
+__all__ = ["CommonDisplay", "DummyDisplay"]
```

Comment:
Do we not need `enable_colab` for end users? I believe this is used in some public notebooks,

Reply:
I don't know what `enable_colab` does. Remove it then?

Reply:
It shouldn't be necessary with 3.0. We shouldn't remove it though - I think we should make it a dummy function that prints out a deprecation warning.